doc/src/site/sphinx/tutorials/hive_support.rst

```diff
@@ -78,7 +78,7 @@ To import data from non-Kerberized Hive, run:
     .. tab-container:: Scala
         :title: Scala
 
-        First, start Sparkling Shell with the  Hive JDBC client jar on the class path
+        First, start Sparkling Shell with the  Hive JDBC client Jar on the class-path
```

Comment:
why `Jar`?

Reply:
Jar is incorrect but should be JAR, thanks